Makefile.doc

```diff
@@ -90,7 +90,7 @@ doc-pdf:\
 doc-ps:\
   doc/stdlib/Library.ps
 
-stdlib: \
+light-stdlib: \
```

Comment:
Why `light`? Shouldn't it be `stdlib-doc` or some such instead?

Reply:
Maybe, here I had to do this a very weird things were happening with the makefile. I am not sure things are still working OK.